ods_ci/utils/scripts/ocm/ocm.py

```diff
@@ -1221,8 +1221,9 @@ def update_ocm_policy(self):
         write_data_in_json(self.update_policies_json, data)
 
         schedule_cluster_upgrade = (
-            "ocm --v={} post /api/clusters_mgmt/v1/clusters/{}/upgrade_policies"
-            " --body {}".format(self.ocm_verbose_level, cluster_id, self.update_policies_json)
+            "ocm --v={} post /api/clusters_mgmt/v1/clusters/{}/upgrade_policies" " --body {}".format(
```

Comment:
```suggestion
            "ocm --v={} post /api/clusters_mgmt/v1/clusters/{}/upgrade_policies --body {}".format(
```
?

Reply:
makes sense; the previous less-than-ideal result of `poetry run ruff format` is most likely a ruff bug, actually